cmd/webhook/main.go

```diff
@@ -83,12 +103,13 @@ func main() {
 	}
 
 	options := webhook.ControllerOptions{
-		ServiceName:    logconfig.WebhookName(),
-		DeploymentName: logconfig.WebhookName(),
-		Namespace:      system.Namespace(),
-		Port:           8443,
-		SecretName:     "eventing-webhook-certs",
-		WebhookName:    "webhook.eventing.knative.dev",
+		ServiceName:       logconfig.WebhookName(),
+		DeploymentName:    logconfig.WebhookName(),
+		Namespace:         system.Namespace(),
+		Port:              8443,
+		SecretName:        "eventing-webhook-certs",
+		WebhookName:       "webhook.eventing.knative.dev",
+		RegistrationDelay: getRegistrationDelayTime() * time.Second,
```

Comment:
adding options to this will cause eventing to not be easily moved to the pkg version of the generic webhook. I recommend not changing the `ControllerOptions` struct. 

/cc @mattmoor

Reply:
We should just be mindful of differences.  Looks like you don't disallow unknown JSON fields, and we do.  You don't have a configmap for defaulting and we do. There are already a bunch.  In this case, I don't see a strong reason to diverge though.

Reply:
The struct `ControllerOptions` has already included the field `RegistrationDelay` in pkg version of the generic webhook https://github.com/knative/pkg/blob/e4bc08cc8deda3eac40a8d53984d09c9d0e1ad48/webhook/webhook.go#L96. Perhaps I'm missing something here, I just init `RegistrationDelay` and doesn't change anything.  @n3wscott